http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -316,6 +316,57 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
+    patch:
+      summary: Updates the configuration of a runtime
+      description: updates a runtime
+      operationId: updateRuntime
+      tags:
+        - runtimes
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: name
+          description: runtimeName. only lowercase alphanumeric characters, numbers and dashes are allowed
+          required: true
+          schema:
+            type: string
+        requestBody:
```

Comment:
I get an indentation error for the above line when I copy/paste the `yaml` content onto http://editor.swagger.io.

Reply:
should be fixed now